style_editor.html

```diff
@@ -10211,6 +10212,16 @@ <h4 class="settings-section-label" title="These settings apply to the whole page
           <div class="settings-section">
             <h4 class="settings-section-label" title="These settings apply to the blade preview above.">Blade Preview:</h4>
 
+            <label title="Toggles the option for swinging around the preview blade above using mouse control.">
+              <input type="checkbox" id="STILL_BUTTON" name="still" onclick="handleClick(this)">
+              <span>Still</span>
```

Comment:
"Still" is not a good description.
How about "disable mouse swings" ?

Reply:
Sure. Several questions then.
"Disable" would probably be assumed if the check box was not on, so what if we invert the checkbox so that ON is mouse swings enabled and that's the default?
Then, "Mouse swings" for the label, or maybe "Swing with mouse moves" ?
Would you want all the relevant code to take on the longer strings too then, where stillState would become mouseSwingState and STILL_BUTTON becomes MOUSE_SWING_BUTTON etc...?

I thought "still" was short and to the point.

Or... just make it "No mouse swings" and be done.

Reply:
ok just changed to Disable mouse swings.